### DISCUSSION TITLE
Feature Request: IR sensor

### DISCUSSION THREAD

**Discussion by bitmvk** [flet-dev/flet#4899] [Extensions]
**Please Describe The Feature To Be Added**

Integration IR sensor

**Suggest A Solution**

https://pub.dev/packages/ir_sensor_plugin

**Comment by NocoleNe** [comment]
I see that you have provided a lot of suggestions, and I personally suggest that platform-specific features should not be added.
Provided by a third party may be better, flet should focus on the basic features, his basic goal, a good UI framework

**Comment by bitmvk** [comment]
I get where you are coming from. But for a UI framework which is meant to work in both IOS and Android, it is required for them to provide platform specific features as there is no other way for the app to access them otherwise, like local authentication and camera, etc...

I know the features I requested are not that important for everyone but the bigger the framework the better
